Makefile

```diff
@@ -166,16 +166,21 @@ examples-dev: ## Generate example output with developer image
 	$(MAKE) cmd/clusterctl/examples/azure/out MANAGER_IMAGE=${DEV_MANAGER_IMAGE}
 
 .PHONY: manifests
-manifests: #cmd/clusterctl/examples/azure/out/credentials ## Generate manifests for clusterctl
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+manifests: cmd/clusterctl/examples/azure/out/credentials ## Generate manifests for clusterctl
```

Comment:
Comment `cmd/clusterctl/examples/azure/out/credentials ## Generate manifests for clusterctl` out.
This was commented before because the target makes the assumption that `clusterazureadm` exists (and it doesn't).